src/common/Input/Input.js

```diff
@@ -36,19 +44,24 @@ const Input = React.forwardRef(
       setInvalid,
       tip,
       type,
+      validationRules: rules,
       value,
       withoutBorder,
       wrapperClassName
     },
     ref
   ) => {
+    ref = useRef()
```

Comment:
I don't think we should reassign the `ref`. We might not be able to use it outside the component.

Reply:
I need the main div ref to use in the useDetectOutsideClick function.
Do you prefer I create another ref?